dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/Contentlet.java

```diff
@@ -88,6 +80,9 @@
    protected Map<String, Object> map = new ContentletHashMap();
    private boolean lowIndexPriority = false;
 
+   private ContentletAPI contentletAPI;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make "contentletAPI" transient or serializable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1948)

Reply:
transient will be ok